frontend/package.json

```diff
@@ -4,7 +4,8 @@
     "build": "nuxt build",
     "dev": "nuxt dev",
     "generate": "nuxt generate",
-    "preview": "nuxt preview"
+    "preview": "nuxt preview",
+    "start": "nuxt build && nuxt preview"
```

Comment:
Is preview the right command for heroku?

Reply:
Yep, preview is what starts up the server. I don't know why they named it preview but that's what it is.